tools/cdhit/cdhit_macros.xml

```diff
@@ -146,14 +146,16 @@
     <conditional name="runtime">
       <param name="tuning" type="select" label="Runtime Memory and Threads">
         <option value="default" selected="true">Use Default settings</option>
-        <option value="tune" >Set Runtime options</option>
+        <option value="options" >Set Runtime options</option>
       </param>
       <when value="default"/>
-      <when value="tune">
-        <param name="threads_num" type="integer" value="1" optional="true" label="number of threads; with 0, all CPUs will be used. (default 1)" >
```

Comment:
Normally, the user must not have access to thread number. You can remove this parameter and replace `threads_num` by `"\${GALAXY_SLOTS:-1}"` in  `<command>`